std_msgs/Header header
float64 q0
float64 q1
float64 q2
float64 q3
float64 tx
float64 ty
float64 tz
float64 roll
float64 pitch
float64 yaw